README.md

```diff
@@ -1,13 +1,67 @@
 # Moxie
 [![Build Status](https://travis-ci.org/DarthStrom/Moxie.svg?branch=master)](https://travis-ci.org/DarthStrom/Moxie)
-
 A tasty mocking library for Swift
 
 ![Moxie](moxie.jpeg)
 
+## Creating mocks
+All mock objects must conform to the Mock protocol and have an instance of Moxie.
+
+```swift
+import Moxie
+
+class MockClass: Mock {
```

Comment:
Maybe we should have most examples use protocols instead of classes to be more Swifty.